server/webapp/WEB-INF/rails.new/app/assets/stylesheets/patterns/forms.scss

```diff
@@ -168,6 +168,9 @@ label.inline {
 }
 
 .form_item_block {
+  &.with-padding-top {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets